project/plugins.sbt

```diff
@@ -1,5 +1,6 @@
 resolvers += Resolver.sonatypeRepo("releases")
 resolvers += Resolver.bintrayRepo("slamdata-inc", "maven-public")
 
-addSbtPlugin("com.eed3si9n"    % "sbt-buildinfo" % "0.9.0")
-addSbtPlugin("com.slamdata"    % "sbt-slamdata"  % "5.1.5")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo"     % "0.9.0")
+addSbtPlugin("com.slamdata" % "sbt-slamdata"      % "5.1.5")
```

Comment:
Is this correct? I thought when we have the `quasar-plugin` we *don't* use `sbt-slamdata`?